### PR TITLE
updates from bug bash

### PR DESCRIPTION
**Pull request by bakerfranke**

### CODE REVIEW

File: dashboard/config/foorm/library/surveys/pd/ayw_module_1.0.json [hunk 1]
```diff
@@ -136,6 +136,7 @@
       {
        "type": "comment",
        "name": "other_activities_m1",
+              "maxLength": 4000,
```
Comment (molly-moen) [inline dashboard/config/foorm/library/surveys/pd/ayw_module_1.0.json]: you can leave this in here but I added a fix that will applies a blanket maximum length for all free responses

File: dashboard/config/foorm/library/surveys/pd/workshop_elements.0.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
         {
          "type": "html",
          "name": "survey_preamble_header_and_text",
-         "html": "<h2>{workshop_course} Academic Year Workshop Post-Survey</h2>\n\n<p><strong>Congratulations! You've completed a Code.org {workshop_course} Academic Year workshop!</strong></p>\n<p>This survey asks questions about your experience today and the workshop as a whole.&nbsp;</p>\n<p><strong>Privacy and Confidentiality</strong></p>\n<ul>\n<li>Your individual responses here are completely <strong>confidential.</strong></li>\n<li>Your identity will&nbsp;<strong>never</strong> be revealed publicly or used to tie any responses back to you personally.</li>\n<li>We may share <strong>de-identified</strong>, aggregate data (representing all responses) publicly.</li>\n<li>Your <strong>de-identified&nbsp;</strong>responses will be used by Code.org, our Regional Partners, and our facilitators for program improvement.</li>\n</ul>\n<p>This survey should take about <strong>10 minutes</strong> to complete.&nbsp;&nbsp;<strong>Your complete honesty and candor are appreciated.</strong></p>"
```
Comment (molly-moen) [inline dashboard/config/foorm/library/surveys/pd/workshop_elements.0.json]: not sure what changed here, but FYI you or @bencodeorg will get a merge conflict depending on who merges first

**Review by molly-moen — APPROVED** [review]

**Comment by bencodeorg** [comment]
There's something wrong with the JSON in the first commit, trying to sort out...

File: dashboard/config/foorm/library/surveys/pd/workshop_elements.0.json (hunk 2 above)
Comment (bencodeorg) [reply dashboard/config/foorm/library/surveys/pd/workshop_elements.0.json]: It was just capitalizing Workshop in Academic Year Workshop, I resolved the merge conflict.

**Pull request merged**